kafka_consumer/tests/test_kafka_consumer.py

```diff
@@ -222,9 +223,8 @@ def test_version_metadata(datadog_agent, kafka_instance, dd_run_check):
     kafka_consumer_check = KafkaCheck('kafka_consumer', {}, [kafka_instance])
```

Comment:
This test should use the env var KAFKA_VERSION instead of using the kafka_client to retrieve the version. But refactoring this is out of scope here.